templates/settings.html

```diff
@@ -0,0 +1,11 @@
+{% extends "global.html" %}
+{% from "_formhelpers.html" import render_field %}
+{% block content %}
+<form method=post action="/settings">
```

Comment:
probably want to place this in a div that more or less centers the form better http://getbootstrap.com/css/#grid
i think ```  <div class="col-md-4 col-md-offset-4"> ``` might be perfect